src/components/designer/taro/actions/MintAssetModal.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { useAsyncCallback } from 'react-async-hook';
+import { Alert, Checkbox, Form, Input, InputNumber, Modal, Select } from 'antd';
+import { SelectValue } from 'antd/lib/select';
+import { usePrefixedTranslation } from 'hooks';
+import { TarodNode } from 'shared/types';
+import { TARO_ASSET_TYPE, TARO_MIN_LND_BALANCE } from 'lib/taro/types';
+import { useStoreActions, useStoreState } from 'store';
+import { MintAssetPayload } from 'store/models/taro';
+import { Network } from 'types';
+
+interface Props {
+  network: Network;
+}
+
+const getSelectedTaroNode = (network: Network) => {
+  const { selected: selectedNode } = useStoreState(s => s.designer.activeChart);
+  return network.nodes.taro.find(node => node.name === selectedNode.id) as TarodNode;
+};
```

Comment:
There seems to be a mix up, I'm looking at my code and I have corrected this. This is removed in my current branch.

Reply:
I think this comment was submitted by accident. I had started the review one day then came back and finished it another day. You updated the commits in between. This was a leftover comment that no longer applied. I didn't even see it in the Github review UI because the file was gone.